packages/py-moose-lib/tests/test_olap_dictionary.py

```
@@ -0,0 +1,746 @@
+
+
+@pytest.mark.parametrize(
+    ("layout", "expected_type"),
+    [
+        (FlatLayout(), "FLAT"),
+        (HashedLayout(), "HASHED"),
+        (SparseHashedLayout(), "SPARSE_HASHED"),
+        (HashedArrayLayout(), "HASHED_ARRAY"),
+        (RangeHashedLayout(), "RANGE_HASHED"),
+        (CacheLayout(size_in_cells=1000), "CACHE"),
+        (SsdCacheLayout(path="/tmp/ssd"), "SSD_CACHE"),
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/ssd"

(S108)